commands/run.js

```diff
@@ -0,0 +1,79 @@
+const cli = require('heroku-cli-util')
+const Sanbashi = require('../lib/sanbashi')
+
+let usage = `
+    ${ cli.color.bold.underline.magenta('Usage:')}
+    ${ cli.color.cmd('heroku container:run web bash')} # Runs bash on the local web docker container
+    ${ cli.color.cmd('heroku container:run worker')}   # Runs the container CMD on the local worker container`
+
+module.exports = function (topic) {
+  return {
+    topic: topic,
+    command: 'run',
+    description: 'builds, then runs the docker image locally',
+    variableArgs: true,
+    help: usage,
+    flags: [
+      {
+        name: 'verbose',
+        char: 'v',
+        hasValue: false
+      },
+      {
+        name: 'arg',
+        hasValue: true,
+        description: 'set build-time variables'
+      }
+    ],
+    run: cli.command(run)
+  }
+}
+
+let run = async function (context, heroku) {
+  if (context.args.length === 0) {
+    cli.error(`Error: Requires one process type\n ${usage} `)
+    process.exit(1)
+  }
+
+  let processType = context.args.shift()
+  let command = context.args
+  let tag = `heroku-local-${processType}`
+  let dockerfiles = Sanbashi.getDockerfiles(process.cwd(), false)
+  let possibleJobs = Sanbashi.getJobs(tag, dockerfiles)
+
+  let jobs = []
+  if (possibleJobs.standard) {
+    possibleJobs.standard.forEach((pj) => { pj.resource = pj.resource.replace(/standard$/, processType)})
```

Comment:
Where does "standard" come from?

Reply:
I have no idea. Looks like it's an array method. I've taken this from the push command.

Reply:
I was curious about `resource.replace(/standard$/, processType)` specifically.

Reply:
Maybe it's copied from another place there too? Can we remove it?

Reply:
@heroku/cli any idea about this?

Reply:
no idea. I looked at the code and it seems that it never gets set from what I can tell.

Reply:
`standard` come from [here](https://github.com/heroku/heroku-container-registry/blob/master/lib/sanbashi.js#L33).
The lib detects the process name part of `Dockerfile.web`, and defaults to `standard` if there is nothing.
Maybe this should be renamed to `default` to be clearer.

Reply:
Wouldn't "web" be better as that's the default process type?

Reply:
`web` works for me. I'll change it.